pkg/controller/plan/adapter/vsphere/builder.go

```diff
@@ -679,6 +690,11 @@ func (r *Builder) mapDisks(vm *model.VM, persistentVolumeClaims []*core.Persiste
 				},
 			},
 		}
+		// For multiboot VMs, if the selected boot device is the current disk,
+		// set it as the first in the boot order.
```

Comment:
`For multiboot VMs, if ...`
otherwise bootDisk is zero, right?

Reply:
yes, otherwise it will keep the order same as we creating and attaching the disks, which means disk 0.